### PR TITLE
Backend testing

### PR DESCRIPTION
**Pull request by rsmith92**
-Made changes in the app.py addJob and deleteJob functions so that they will work correctly
-Also made changes in mongo.py so that it works with app.py

### CODE REVIEW

File: flask_backend/app.py [hunk 1]
```diff
@@ -2,81 +2,151 @@
...
+ 
+def addJob(): 
+   jobToAdd = request.get_json()
+   jobs['job_list'].append(jobToAdd) 
+   # userToAdd['id'] = gen_random_id()     # check for duplicate before appending.. todo
+   # make DB request to add user
+   newJob = Job(jobToAdd)
+   newJob.save()
+   resp = jsonify(newJob), 201
+   return resp
```
Comment (mgray07) [inline flask_backend/app.py:65]: LGTM

File: flask_backend/app.py [hunk 2]
```diff
@@ -2,81 +2,151 @@
...
+def deleteJob():
+   jobToDelete = request.get_json()
+   resp = jobToDelete.remove()            # DOUBLE CHECK THAT THIS WORKS                  
+   #print(f'resp = {resp}')
+   if resp['n'] == 1:                     # WHAT DOES THIS DO?????
+      return {}, 204
+   #if user.reload() :
+   #   return user
+   else :
+      return jsonify({"error": "job not found"}), 404
```
Comment (mgray07) [inline flask_backend/app.py:77]: Maybe figure out why line 71 wasnt working

Comment (rsmith92) [reply flask_backend/app.py:77]: I think it is a completely unnecessary line that was used for testing in the previous file that I got it from.

File: flask_backend/mongo.py [hunk 3]
```diff
@@ -28,29 +38,44 @@ def reload(self):
...
+#        for job in jobs:
+#           job["_id"] = str(job["_id"])
+        return jobs
+   
+
+    def find_by_status(self, status):
+        jobs = list(self.collection.find({"status": status}))
+#        for job in jobs:
+#           job["_id"] = str(job["_id"])
+        return jobs
```
[the remark this answers is not in the document]
Comment (rsmith92) [reply flask_backend/mongo.py:75]: I don't believe that this is necessary for the functionality of the project.

**Pull request merged**